glide.yaml

```diff
@@ -21,7 +21,8 @@ import:
 - package: github.com/go-ini/ini
   version: ^1.21.0
 - package: github.com/projectcalico/libcalico-go
-  version: v1.1.1
+  repo: https://github.com/fasaxc/libcalico-go.git
+  version: 3428e33dd83712164f8b46b576c91098f7cf9b36
```

Comment:
As discussed, was worried that glide didn't freshen the branch.  It's now been merged so I an pin to master now.